src/plugins/discover/public/embeddable/saved_search_embeddable.tsx

```diff
@@ -391,15 +392,17 @@ export class SavedSearchEmbeddable
       Array.isArray(searchProps.columns)
     ) {
       ReactDOM.render(
-        <FieldStatsTableSavedSearchEmbeddable
-          services={searchProps.services}
-          indexPattern={searchProps.indexPattern}
-          columns={searchProps.columns}
-          savedSearch={this.savedSearch}
-          filters={this.input.filters}
-          query={this.input.query}
-          onAddFilter={searchProps.onFilter}
-        />,
+        <KibanaThemeProvider theme$={searchProps.services.core.theme.theme$}>
+          <FieldStatsTableSavedSearchEmbeddable
+            services={searchProps.services}
+            indexPattern={searchProps.indexPattern}
+            columns={searchProps.columns}
+            savedSearch={this.savedSearch}
+            filters={this.input.filters}
+            query={this.input.query}
+            onAddFilter={searchProps.onFilter}
+          />
+        </KibanaThemeProvider>,
```

Comment:
@majagrubic in this file, Line 416 there seems to be another ReactDOM.render that needs to be adapted